st2common/tests/unit/test_unit_testing_mocks.py

```diff
@@ -0,0 +1,87 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+
+from st2tests.base import BaseSensorTestCase
+from st2tests.mocks.sensor import MockSensorWrapper
+from st2tests.mocks.sensor import MockSensorService
+
+__all__ = [
+    'BaseSensorTestCaseTestCase',
+    'MockSensorServiceTestCase'
+]
+
+
+class MockSensorClass(object):
+    pass
+
+
+class BaseSensorTestCaseTestCase(BaseSensorTestCase):
```

Comment:
We could also move pack related test case classes to a different module (e.g. `st2tests.packs.BaseSensorTestCase`).

What do others think?